llvm/unittests/Transforms/Vectorize/VPlanVerifierTest.cpp

```diff
@@ -387,8 +389,6 @@ TEST_F(VPIRVerifierTest, testVerifyIRPhiInExitVPIRBB) {
                         {HeaderBlock->front().getVPSingleValue()});
   DefI->insertBefore(Plan->getMiddleBlock()->getTerminator());
   Plan->getExitBlocks()[0]->front().addOperand(DefI);
-  VPValue *Zero = Plan->getConstantInt(32, 0);
-  Plan->getScalarHeader()->front().addOperand(Zero);
```

Comment:
The initial plan now already has the operands of the scalar header phis set up correctly, so this change is not needed any more to fix them up.